tfx/components/example_gen/csv_example_gen/executor.py

```diff
@@ -31,35 +31,52 @@
 from tfx.utils import io_utils
 
 
+def _int_handler(cell: bytes) -> tf.train.Feature:
```

Comment:
is bytes or csv_decoder.CSVCell? check line 93 & 99

Reply:
i printed out the type of cell inside the method and it was 'bytes', if it's type csv_decoder.CSVCell, it should print out that?

Reply:
check tfx_bsl/coders/csv_decoder.py 
csv_decoder.CSVCell = bytes,

let use either cell:CSVCell or value:bytes

Reply:
i see, i used cell: CSVCell